doc/source/whatsnew/v1.4.1.rst

```diff
@@ -16,6 +16,7 @@ Fixed regressions
 ~~~~~~~~~~~~~~~~~
 - Regression in :meth:`Series.mask` with ``inplace=True`` and ``PeriodDtype`` and an incompatible ``other`` coercing to a common dtype instead of raising (:issue:`45546`)
 - Regression in :meth:`DataFrame.loc.__setitem__` losing :class:`Index` name if :class:`DataFrame` was empty before (:issue:`45621`)
+- Fixed failing join operations with overlapping ``IntervalIndex`` (:issue:`45661`)
 -
```

Comment:
Regression in :func:`join` with overlapping :class:``IntervalIndex`` raising an ``InvalidIndexError``


We normally write regression fixes like that

Reply:
Thanks